Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -17,7 +22,7 @@ class InAppMessageViewTest: UnitTest {
 
         DIGraphShared.shared.override(value: queueMock, forType: MessageQueueManager.self)
         DIGraphShared.shared.override(value: engineProvider, forType: EngineWebProvider.self)
-        DIGraphShared.shared.override(value: inlineMessageDelegateMock, forType: InlineMessageDelegate.self)
+        DIGraphShared.shared.override(value: inlineMessageDelegateMock, forType: InAppMessageViewActionDelegate.self)
```

Comment:
```suggestion
```

I don't think this is needed. The delegate is set on the View instances in the test functions, not injected through DI.